packages/typespec-test/test/versioning/spec/tspconfig.yaml

```diff
@@ -0,0 +1,14 @@
+emit:
+  - "@azure-tools/typespec-ts"
+options:
+  "@azure-tools/typespec-ts":
+    azureSdkForJs: false
+    isModularLibrary: false
+    experimental: true
```

Comment:
what should we generate if we don't pass experimental and the typespec is multi api version ?

Reply:
I wonder if we should generate api version as client constructor parameter in that case?

Reply:
I think we should generate the latest version if it's not experimental and the typespec is multi api version ? I remember previously, we had an issue that if an operation is removed in the latest version, we still generate this operation https://github.com/Azure/typespec-azure/issues/129

Reply:
For the scope of this PR we'll keep generating what we generate today. The superset of TypeSpec, with no regard to versioning. We should revisit this as we approach the completion of this feature